src/test/ipv-stub/ipv-authorize.test.ts

```diff
@@ -0,0 +1,38 @@
+
+describe("IPV Authorize", () => {
+  const api = supertest("http://127.0.0.1:3000/");
+
+  it("should return 200 for valid GET request", async () => {
+    const response = await api
+      .get("/authorize")
+      .set("Authorization", "Bearer " + ENCRYPTED_JWT);
+    expect(response.statusCode).toBe(200);
+  });
+
+  it("should return 302 for valid POST request", async () => {
+    const response = await api
+      .post("/authorize")
+      .set("Authorization", "Bearer " + ENCRYPTED_JWT);
+    expect(response.statusCode).toBe(302);
+    expect(response.headers.location).toBe(
+      "https://oidc.sandpit.account.gov.uk/ipv-callback?code=12345"
+    );
+  });
+
+  it("should return 400 if token is not present in authorize request", async () => {
+    const response = await api.get("/authorize");
+    expect(response.statusCode).toBe(400);
+  });
+
+  it("should return 500 for invalid token in request", async () => {
```

Comment:
Since we have specific 500 errors we should test those errors.

Reply:
Leaving for now - something for unit test as the integration tests don't have to test this. New ticket/ review after stubs are done.